uploader.v2.json

```diff
@@ -388,6 +388,11 @@
             "pattern": [],
             "symbolic_name": "ls_var_spool_clientmq"
         },
+        {
+             "command": "/bin/ls -l /var/lib/cni/networks/openshift-sdn",
+             "pattern": [],
+             "symbolic_name": "ls_var_lib_cni_networks_openshift-sdn"
```

Comment:
Same comment as above.

Reply:
@bfahr Thanks for your review. I have changed the code according to your comments. Could you please help to review the changes?